scripts/install.gs

```diff
@@ -18,6 +18,16 @@ if *goGet {
 	Printfln("go get -u -v %s/%s", GCSE, a)
 	MustSucc(Bash("go get -u -v %s/%s", GCSE, a))
   }
+
+  additionalPackages := []string{
```

Comment:
This is weird and getting the dependencies explicitly seems not an optimal solution. I'll have a test later.